README.md

```diff
@@ -5,31 +5,39 @@ This repo is linked to main BGNN Fish project rpo [BGNN_Snakemake](https://githu
 
 ## 1 - Short description
 
-  + We use raw fish image [available at Tulane](http://www.tubri.org/HDR/INHS/) and Metadata json file generated by [Drexel_metadata generator] (https://github.com/hdr-bgnn/drexel_metadata) to crop the fish out of a more complexe image.
-  + The metadata json file contained the bounding box of the fish which use to crop the fish [here is an exmaple](https://github.com/hdr-bgnn/Crop_image/blob/main/Test_data/INHS_FISH_000742_cropped.jpg). We increase the size of the bounding box around the fish by 10% increase in case the bounding box (from the metadata) would truncate the edge of the fish. The Cropped image will be used later by a segmentation algorithm such as [trait_segmenation](https://github.com/hdr-bgnn/BGNN-trait-segmentation)
+  + We use raw fish image [available at Tulane](http://www.tubri.org/HDR/INHS/) and Metadata json file generated by [Drexel_metadata_rformatter] (https://github.com/hdr-bgnn/drexel_metadata_formatter) to crop the fish out of a more complexe image.
```

Comment:
I don't think the reformatter generates the metadata JSON, it only reformats it. Isn't that so?

Reply:
you are right.
I change the text  to
We use raw fish image [available at Tulane](http://www.tubri.org/HDR/INHS/) and Metadata json file generated by [Drexel_metadata](https://github.com/hdr-bgnn/drexel_metadata) and reformatted by [Drexel_metadata_formatter](https://github.com/hdr-bgnn/drexel_metadata_formatter) to crop the fish out of a more complexe image.